fileinfo.go

```diff
@@ -61,6 +63,16 @@ func (f *FileInfo) Group() string {
 	return f.group
 }
 
+// Digest is the md5sum of a file in a RPM package
+func (f *FileInfo) Digest() string {
+	return f.digest
+}
+
+// LinkTarget is the link target of a link file in a RPM package
+func (f *FileInfo) LinkTarget() string {
```

Comment:
Small nit pick: would you consider renaming this to `Linkname`? Purely to keep in the theme of other packages like `archive/tar`.

Reply:
Sure thing, I've modified the last commit and pushed.